docs/source/verdi/verdi_user_guide.rst

```diff
@@ -355,12 +355,7 @@ Manage ``Data`` nodes.
 ---------------
 Commands intended for developers, such as setting :doc:`config properties<properties>` and running the unit test suite.
 
-  * **delproperty**: delete a property from the configuration
```

Comment:
Are the new command line verdi commands described? (couldn't see the change)

Reply:
I have added the new command to the `verdi` docs